src/index.js

```diff
@@ -0,0 +1,45 @@
+// import _ from 'lodash';
+// import printMe from './print.js';
+import './style.css';
+
+// import _ from 'lodash';
+// import './style.css';
```

Comment:
In order to have a more concise code base, kindly remove commented pieces of code.